src/nimgen/gencore.nim

```diff
@@ -20,7 +20,7 @@ proc addEnv*(str: string): string =
     discard
 
   # if there are still format args, print a warning
-  if newStr.contains("${"):
+  if newStr.contains("$") and not newStr.contains("$replace("):
     echo "WARNING: \"", newStr, "\" still contains an uninterpolated value!"
 
```

Comment:
I'm wondering if this should be an error. If an env. var is not declared then we should stop and advise the user to refer to the docs for the wrapper to learn about the var.

Reply:
This isn't an error because there might be values that just have a dollar sign in them and aren't meant to be interpolated. I should change this to "might contain an interpolated value"